conf/app.routes

```diff
@@ -8,7 +8,7 @@ GET        /start   uk.gov.hmrc.plasticpackagingtax.registration.controllers.Sta
 
 GET        /register-plastic-packaging-tax                  uk.gov.hmrc.plasticpackagingtax.registration.controllers.RegistrationController.displayPage()
 
-GET        /incorp-id-callback      uk.gov.hmrc.plasticpackagingtax.registration.controllers.IncorpIdController.incorpIdCallback(journeyId)
+GET        /incorp-id-callback      uk.gov.hmrc.plasticpackagingtax.registration.controllers.GrsController.grsCallback(journeyId)
```

Comment:
Should the callback url change to make it more "generic" as well?

Reply:
I think it should. I'll make it so.